package.json

```diff
@@ -22,17 +35,21 @@
     "sql-formatter": "^15.0.2",
     "swr": "^2.2.4",
     "tailwind-merge": "^2.2.0",
-    "typescript": "^4.4.2",
     "web-vitals": "^2.1.0",
     "zustand": "^4.4.7"
   },
   "scripts": {
     "start": "vite",
-    "build": "tsc && vite build",
+    "build": "yarn build-app && yarn build-lib",
```

Comment:
Currently the Dockerfile calls `yarn build`, do we need it to actually call `yarn build-lib` too? Should we change it to just do `build-app`?